state/state.go

```diff
@@ -145,6 +146,11 @@ func (st *State) ControllerTag() names.ControllerTag {
 	return st.controllerTag
 }
 
+func (st *State) ControllerTimestamp() (*time.Time, error) {
```

Comment:
What this function does is completely obvious good sir, yet still, perhaps we should comment it since it is a public method. Or perhaps it would be better to comment it on the `backend` interface?